tests/drivers/gpio/gpio_basic_api/src/test_callback_trigger.c

```diff
@@ -68,8 +69,6 @@ static int test_callback(int mode)
 		goto err_exit;
 	}
 
-	struct drv_data *drv_data = &data;
```

Comment:
What's the warning here?   I ask, because if the answer is "inline declaration", then I've got some rework to do because I do that all the time...